### DISCUSSION TITLE
The Almighty Loaf

### DISCUSSION THREAD

**Discussion by jmcrisostomo** [warengonzaga/css-text-portrait-builder#41] [Showcase]
I didn't know you can build a CSS Text Portrait at a whole new level.

![1234-orange-hookworm-5zogwqfw ws-us17 gitpod io_](https://user-images.githubusercontent.com/25568550/143256940-a8f9e83b-ca79-489e-ae82-2e8239616f46.png)

Thank you for this. ⭐

Inspired by the memes. 🤣
https://www.youtube.com/watch?v=VAC-5BQnuXI

**Comment by warengonzaga** [comment]
Bruh @jmcrisostomo this is real art lol!
